dbschema/models.py

```diff
@@ -13,3 +13,4 @@ class Users(Base):
     location = Column(String)
     gender = Column(String)
     dob = Column(Date)
+    PA_evaluation = Column(Integer)
```

Comment:
Can you add a comment that this is temporary? This will be removed with https://github.com/PerfectFit-project/virtual-coach-server/issues/125